apps/api/src/services/external/githubService.ts

```diff
@@ -22,19 +22,25 @@ export function getOctokit() {
 export const getProviderAttributesSchema = async (): Promise<ProviderAttributesSchema> => {
   // Fetching provider attributes schema
   const response = await cacheResponse(
-    30,
+    60 * 5,
```

Comment:
```typescript
import { secondsInMinute } from "date-fns/constants";

5 * secondsInMinute
```

Reply:
I switched to using `date-fns/minutesToSeconds`